src/crypto-verifier.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {getService} from './service';
```

Comment:
Sorry, I'm not following the implications of your dependency concern.

I'm happy to put this inside A4A, and not make it a service, if that's what you'd like. If anyone ever needs it for anything else, we can deal with it at that time.

If you're suggesting something else, can you explain?